CorsixTH/Lua/cheats.lua

```diff
@@ -35,15 +35,13 @@ function Cheats:Cheats(hospital)
   -- Cheats to appear specifically in the cheats window
   -- New cheats require an afterLoad when added
   self.cheat_list = {
+    {name = "epidemic",       toggle = self.cheatToggleEpidemic,  func = self.cheatEpidemic, state = self.hospital.epidemics_disabled},
+    {name = "earthquake",     toggle = self.cheatToggleEarthquake, func = self.cheatEarthquake, state = self.hospital.earthquakes_disabled},
     {name = "money",          func = self.cheatMoney},
     {name = "all_research",   func = self.cheatResearch},
     {name = "emergency",      func = self.cheatEmergency},
-    {name = "epidemic",       func = self.cheatEpidemic},
-    {name = "toggle_epidemic", func = self.cheatToggleEpidemic},
     {name = "toggle_infected", func = self.cheatToggleInfected},
```

Comment:
Now that toggle is being used in this context as an on/off switch, toggle_infected is - not a good name.